convert/convert_test.go

```diff
@@ -5,16 +5,17 @@ import (
 	"github.com/vertgenlab/gonomics/fasta"
 	"github.com/vertgenlab/gonomics/vcf"
 	"testing"
+	"strings"
 )
 
 var seqA []dna.Base = dna.StringToBases("--TTTC--ATGAATAA")
 var seqB []dna.Base = dna.StringToBases("CCATTCCAA--CAGA-")
 var inputFa []*fasta.Fasta = []*fasta.Fasta{{"eggplant", seqA}, {"squash", seqB}}
-var var1 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 1, Id: ".", Ref: "T", Alt: "A", Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."}
-var var2 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 4, Id: ".", Ref: "C", Alt: "CCA", Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."}
-var var3 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 5, Id: ".", Ref: "ATG", Alt: "A", Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."}
-var var4 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 8, Id: ".", Ref: "A", Alt: "C", Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."}
-var var5 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 10, Id: ".", Ref: "T", Alt: "G", Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."}
+var var1 vcf.Vcf = vcf.Vcf{Chr: "chr1", Pos: 1, Id: ".", Ref: "T", Alt: strings.Split("A", ","), Qual: 100.0, Filter: "PASS", Info: "."}
```

Comment:
Not sure if this is the best way to handle the alt. Another "type" of Vcf you can have is a "structure variance" vcf file and that is not separated by commas. See page pages 12-18 [https://samtools.github.io/hts-specs/VCFv4.2.pdf](https://samtools.github.io/hts-specs/VCFv4.2.pdf)

Reply:
I think the structure variance alleles are still comma separated.  You can see what I believe is an example at the bottom of section 5.4.3 in the document linked above.

Reply:
I think we should table this issue for now until someone has need of a structure variance allele parser, at which point it makes sense to rework some of this.